src/components/Dashboard/Dashboard.jsx

```diff
@@ -14,19 +14,24 @@ import SearchBar from "material-ui-search-bar";
 // import LinearProgress from "@material-ui/core/LinearProgress";
 import { useStyleDashboard } from "./Widgets/Style";
 import FiltersFlat from "./Widgets/FiltersFlat";
-import { ProfileContext } from '../../profileContext' 
-
+import { ProfileContext } from "../../profileContext";
+import { db } from "../../firebase";
+import CircularProgress from "@material-ui/core/CircularProgress";
 
 export default function Dashboard() {
-  const [profile, ] = useContext(ProfileContext)
-
+  const [profile] = useContext(ProfileContext);
+  const [posts, setPosts] = useState();
   const [state, setState] = React.useState({
     mobileView: false,
     mediumView: false,
   });
 
   const { mobileView, mediumView } = state;
-
+  React.useEffect(() => {
+    db.collection("posts").onSnapshot((snapshot) => {
+      setPosts(snapshot.docs.map((doc) => doc.data()));
+    });
+  }, []);
```

Comment:
Where is the unsubscribe method returned from Firebase to clear things up?

Reply:
Fixed 140ca9f9a8e00f3dd1f9e0803e6fff7bbca30740